packages/swap/src/routes/swap/index.ts

```diff
@@ -35,7 +35,8 @@ type SwapWithBroadcast = Swap & {
     | null;
 };
 
-const uuidRegex = /^[a-f\d]{8}(-[a-f\d]{4}){4}[a-f\d]{8}$/i;
+const channelIdRegex = /^(\d+)-([a-z]+)-([a-z]+)$/i;
```

Comment:
channel id should be a number
```suggestion
const channelIdRegex = /^(\d+)-([a-z]+)-(\d+)$/i;
```